cvat/apps/engine/views.py

```diff
@@ -189,6 +189,9 @@
 _DATA_UPDATED_DATE_HEADER_NAME = 'X-Updated-Date'
 _RETRY_AFTER_TIMEOUT = 10
 
+def get_logo_uri(request) -> str:
+    return request.build_absolute_uri(f'{settings.STATIC_URL}{settings.LOGO_FILENAME}')
```

Comment:
```suggestion
    return request.build_absolute_uri(storages["staticfiles"].url(settings.LOGO_FILENAME))
```

This will correctly handle URL escaping and concatenation.

Reply:
Updated